space/internal/components/migration/readonlyfixer/relationsfixer.go

```diff
@@ -0,0 +1,91 @@
+package readonlyfixer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+
+	"github.com/anyproto/anytype-heart/core/block/editor/basic"
+	"github.com/anyproto/anytype-heart/core/block/editor/smartblock"
+	"github.com/anyproto/anytype-heart/pkg/lib/bundle"
+	"github.com/anyproto/anytype-heart/pkg/lib/database"
+	"github.com/anyproto/anytype-heart/pkg/lib/logging"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+	"github.com/anyproto/anytype-heart/space/internal/components/migration/common"
+	"github.com/anyproto/anytype-heart/util/pbtypes"
+)
+
+const name = "ReadonlyRelationsFixer"
```

Comment:
better to have names like CName, or the other approach - you can actually pass log to migration from parent component

Reply:
I have added log as input parameter of Migration.Run, so migration name is displayed as zap string field of log message